### PR TITLE
feat: Bump velero to 3.1.2

### PR DESCRIPTION
**Pull request by gracedo**
**What problem does this PR solve?**:
For some reason, velero isn't getting bumped automatically. Bumping here to latest v3.1.2

**Which issue(s) does this PR fix?**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

**Comment by gracedo** [comment]
https://github.com/mesosphere/kommander/pull/2924 i ✅

**Review by sonofgibs — APPROVED** [review]

**Review by cbuto — APPROVED** [review]

File: services/velero/3.4.0/velero.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ spec:
   wait: true
   interval: 6h
   retryInterval: 1m
-  path: ./services/velero/3.3.1/helmrelease
```
Comment (cbuto) [inline services/velero/3.4.0/velero.yaml:16]: there is also a pre-install kustomization that i don't see in this PR, maybe it got removed with conflicts or something

Comment (gracedo) [reply services/velero/3.4.0/velero.yaml:16]: oh shoot! lemme fix that..

Comment (gracedo) [reply services/velero/3.4.0/velero.yaml:16]: https://github.com/mesosphere/kommander-applications/pull/947/commits/bf567e6ba87ac9686425fb0be864558fb8e895a3 think this'll fix it

**Review by cbuto — APPROVED** [review]

**Pull request merged**